components/hooks/useComponentComunication/__tests__/stories.js

```diff
@@ -1,27 +1,28 @@
 /**
- * Random Color
+ * Component Comunication Service
  */
 
 // React
-import { useState, useRef } from 'react'
+import React, { useState, useRef } from 'react'
```

Comment:
@mohsenParvar react import should be deleted? were you getting lint errors on this ?